browser-extension/manifest.json

```
@@ -1,20 +1,21 @@
 {
   "name": "Tournesol Extension",
-  "version": "2.0.2",
+  "version": "2.0.3",
   "description": "Open Tournesol directly from Youtube",
   "permissions": ["storage", "https://tournesol.app/", "https://api.tournesol.app/", "https://www.youtube.com/", "contextMenus", "activeTab", "webNavigation"],
   "manifest_version": 2,
   "icons": {
-    "64": "favicon64.png",
-    "512": "favicon512.png"
+    "64": "Logo64.png",
+    "128": "Logo128.png",
+    "512": "Logo512.png"
   },
   "background": {
     "page": "background.html",
     "persistent": true
   },
   "browser_action": {
     "default_icon": {              
-      "16": "favicon.png"
+      "16": "Logo16.png"
```

It would be useful to define a larger alternative here, for high-dpi displays:
```suggestion
      "16": "Logo16.png",
      "64": "Logo64.png"
```